apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -0,0 +1,144 @@
+                  <td>
+                    ${needToSync
+                      ? resLocals.authz_data.has_course_permission_edit &&
+                        !resLocals.course.example_course
+                        ? html`
+                            <span class="text-danger">
+                              You must
+                              <a href="<%= urlPrefix %>/<%= navPage %>/syncs">sync your course</a>
+                              before viewing or editing its configuration
+                            </span>
+                          `
+                        : html`
+                            <span class="text-danger">
+                              A course editor must sync this course before anyone can view or edit
+                              its configuration</span
+                            >
+                          `
+                      : ''}
+                    ${noInfo
+                      ? html`<span class="text-danger">Missing configuration file</span>
+                          ${resLocals.authz_data.has_course_permission_edit &&
+                          !resLocals.course.example_course
+                            ? html`
+                                <form name="add-configuration-form" class="d-inline" method="POST">
+                                  <input
+                                    type="hidden"
+                                    name="__csrf_token"
+                                    value="${resLocals.__csrf_token}"
+                                  />
+                                  <button
+                                    name="__action"
+                                    value="add_configuration"
+                                    class="btn btn-xs btn-secondary mx-2"
+                                  >
+                                    <i class="fa fa-edit"></i>
+                                    <span class="d-none d-sm-inline">Create infoCourse.json</span>
+                                  </button>
+                                </form>
+                              `
+                            : ''} `
+                      : ''}
+                    ${!needToSync && !noInfo
+                      ? resLocals.authz_data.has_course_permission_view
+                        ? html`
+                            <a
+                              href="${resLocals.urlPrefix}/${resLocals.navPage}/file_view/infoCourse.json"
+                            >
+                              infoCourse.json
+                            </a>
+                            ${resLocals.authz_data.has_course_permission_edit &&
+                            !resLocals.course.example_course
+                              ? html`
+                                  <a
+                                    tabindex="0"
+                                    class="btn btn-xs btn-secondary mx-2"
+                                    role="button"
```

Comment:
This is a link, not a button, let's not lie to accessibility tools!

Reply:
Removed `role="button"` in [91314d4](https://github.com/PrairieLearn/PrairieLearn/pull/9406/commits/91314d4c3e75e28d2bda3cfe367f712392d9d690).